src/Common/Ian/Ian_Client.php

```diff
@@ -0,0 +1,386 @@
+<?php
+/**
+ * Handles IAN Client setup and actions.
+ *
+ * @since   TBD
+ *
+ * @package TEC\Common\Ian
+ */
+
+namespace TEC\Common\Ian;
+
+use Tribe__Main;
+use TEC\Common\Telemetry\Telemetry;
+use TEC\Common\Admin\Conditional_Content\Dismissible_Trait;
+
+/**
+ * Class Ian_Client
+ *
+ * @since   TBD
+
+ * @package TEC\Common\Ian
+ */
+final class Ian_Client {
```

Comment:
This class is doing too much, lets the individual Notifications into a class that extends the Dismissable Trait.